frontend/src/sass/components/_forest-title.scss

```diff
@@ -0,0 +1,22 @@
+.forest-title{
```

Comment:
I only looked at the styles for `.forest-title` since we're not currently using the dropdown.

1. There seems to be more css here than necessary, in particular, i'm not sure the following have any effect:
-  the `!important` in `text-align`
- `text-align-last` (a new one for me :))
- `direction`
- `min-width`
- `background-position`

2. colors and fonts should be coming directly from our 'theme' and if we are adding new ones, then they should be added globally and be available as a SASS variable. For example, the `font-family` here should just use `$font-serif` as defined in the USWDS SASS. I will add a comment to the issue regarding the new color, I'm not sure of the value of adding it since I can't even tell the difference to our default black.

3. I would prefer to using `em`s or `rem`s for items such as `line-height` IF we don't have variables or utility classes that work, and it seems reasonable.

Reply:
Correct. The additional CSS was residual shrapnel from styling the dropdown, which I should have placed in #forest-title-drop instead of extending from forest-title. I will remove for now until we decide if we're doing a dropdown or not.

Reply:
Waiting for response from Designers about h2 / h1, will push that change once they reply.